free-podcasts-screencasts-es.md

```diff
@@ -43,7 +43,7 @@
 * [La Buhardilla Geek](https://www.ivoox.com/podcast-buhardilla-geek_sq_f1465450_1.html) (podcast)
 * [Maestría JS](https://anchor.fm/maestriajs) (podcast) (Última Actualización, Mayo 2020)
 * [Programar es una Mierda](https://www.programaresunamierda.com) (podcast)
-* [The Dojo MX Livestreams](https://www.youtube.com/playlist?list=PLfeFnTZNTVDMo5ZljjOZu0FquW6f9hqst)(screencast)
+* [The Dojo MX Livestreams](https://www.youtube.com/playlist?list=PLfeFnTZNTVDMo5ZljjOZu0FquW6f9hqst)(screencast) - Pláticas con expertos en programación, desarrollo, arquitectura de sistemas, liderazgo técnico en español.
```

Comment:
Hey @hectorip! 

- The title of the playlist is called "Livestreams" so that should go in the [brackets].
- It would be after "La Buhardilla Geek" since Livestreams comes after La ... but before "Maestría JS"

- The playlist link is perfect! The author of the YouTube channel is "The Dojo MX" so it should come after the playlist link with a space and "- The Dojo MX" 
- Following this would be a space and "(screencast)" 
- Even further, you can follow it with extra information in ()

- I have commented below how I think it would look like so that the Travis CI build can pass!


```suggestion
* [La Buhardilla Geek](https://www.ivoox.com/podcast-buhardilla-geek_sq_f1465450_1.html) (podcast)
* [Livestreams](https://www.youtube.com/playlist?list=PLfeFnTZNTVDMo5ZljjOZu0FquW6f9hqst) - The Dojo MX (screencast) (Pláticas con expertos en programación, desarrollo, arquitectura de sistemas, liderazgo técnico en español)
* [Maestría JS](https://anchor.fm/maestriajs) (podcast) (Última Actualización, Mayo 2020)
* [Programar es una Mierda](https://www.programaresunamierda.com) (podcast)
```